vespaclient-java/src/main/java/com/yahoo/vespavisit/VdsVisit.java

```diff
@@ -348,6 +348,13 @@ protected static Options createOptions() {
                         FixedBucketSpaces.defaultSpace(), FixedBucketSpaces.globalSpace(), FixedBucketSpaces.defaultSpace()))
                 .build());
 
+        // TODO Vespa 9 replace with --tensor-long-form ?
+        options.addOption(Option.builder()
+                .longOpt("tensor-short-form")
```

Comment:
Consider not using dash in parameter name (the other parameters don't)

Reply:
I considered it, but I think writingoptionslikethisispointlesslyhardtoread so I went for the "be the change you want to see" approach. Save for entirely replacing the tool it's not given how we can ever transition away from a (IMHO) very suboptimal naming strategy if it cannot be done gradually.

But it might be less horrible if I rename it to something like `--shorttensors` instead, so I think I'll go with that.

Reply:
Renamed